### PR TITLE
Color vocab

### PR DESCRIPTION
**Pull request by azaroth42**
Add RGB colorspace and some primary colors to instance table in vocab.

### CODE REVIEW

File: cromulent/vocab.py [hunk 1]
```diff
@@ -324,6 +324,7 @@ def register_instance(name, data):
 	"partsUnit": {"parent": MeasurementUnit, "id": "300404159", "label": "parts"},
 	"pageCount": {"parent": MeasurementUnit, "id": "300194222", "label": "pages"},
 	"pixels": {"parent": MeasurementUnit, "id": "300266190", "label": "pixels"},
+	"rgb_colorspace": {"parent": MeasurementUnit, "id": "300266239", "label": "rgb"},
```
Comment (kasei) [inline cromulent/vocab.py]: Out of curiosity, how is this used in practice where the values aren't scalar?

Comment (azaroth42) [reply cromulent/vocab.py]: I just wrote a big implementation note about that exact issue!  As Dimension's `value` takes a number (for obvious reasons) the typical hex string for RGB doesn't work.
CRM doesn't allow Dimensions to be partitioned (so no Feet, Inches, Lines; Red, Green, Blue; Pounds, Shillings, Pence) ... so everything has to be decimalized :(

So:

```
{
      "id": "https://linked.art/example/value/8", 
      "type": "Dimension", 
      "_label": "Red", 
      "value": 16711680, 
      "unit": {
        "id": "http://vocab.getty.edu/aat/300266239", 
        "type": "MeasurementUnit", 
        "_label": "rgb"
      }, 
      "classified_as": [
        {
          "id": "http://vocab.getty.edu/aat/300056130", 
          "type": "Type", 
          "_label": "Color"
        }
    ]
  }
```

Comment (kasei) [reply cromulent/vocab.py]: What is `16711680`? The hex value of the integer obtained from the adjacent bytes of 8bpc?

Comment (azaroth42) [reply cromulent/vocab.py]: Yep :( 
```
>>> int('0xFF0000', 16)
16711680
```

Comment (kasei) [reply cromulent/vocab.py]: Yikes. Seems like a lot of assumptions baked into that. Is there a pattern for cases where a vector is really needed for the content value? Or for cases where you might have mixed color depths? Or do you just need to find a way to pack it into a scalar, and differentiate with different `MeasurementUnit`s?

Comment (azaroth42) [reply cromulent/vocab.py]: All the alternatives that we could find are worse. The [issue](https://github.com/linked-art/linked.art/issues/191) is still open, but had to document something to move forward with.

**Pull request merged**